cve_bin_tool/version_scanner.py

```diff
@@ -33,15 +36,16 @@ class VersionScanner:
     """ "Scans files for CVEs using CVE checkers"""
 
     CHECKER_ENTRYPOINT = "cve_bin_tool.checker"
+    Infogen = Iterator[Tuple[Union[ProductInfo, None], str]]
```

Comment:
This is not very self explanatory.  It needs documentation, or perhaps we need some appropriately named custom types to make this clearer.

Reply:
I will make it a custom type like `ProductInfo`, hope that makes it clearer.